frontend/src/scenes/funnels/funnelUtils.test.ts

```diff
@@ -30,3 +30,72 @@ describe('getMeanAndStandardDeviation', () => {
         })
     })
 })
+
+describe('getBreakdownStepValues()', () => {
+    it('is baseline breakdown', () => {
+        expect(getBreakdownStepValues({ breakdown: 'blah', breakdown_value: 'Blah' }, 21, true)).toStrictEqual({
+            rowKey: 'baseline_0',
+            breakdown: ['baseline'],
+            breakdown_value: ['Baseline'],
+        })
+    })
+    it('breakdowns are happy arrays', () => {
```

Comment:
what's a "happy array"? I've not seen the term before...

Reply:
Haha I should probably call this "well formed"

Reply:
Ah, I see. No, I love "happy" I'm going to start using it.

Knew someone who called braces, brackets, etc squares, smooths, and squiggles which has stuck with me as the easiest way to distinguish between them :)